### ISSUE TITLE
Running more printers on the same Pi

### ISSUE THREAD

**Issue by guysoft** [guysoft/OctoPi#5]
OctoPi can support multiple instances of OctoPrint. There should be a simple way to create new instances.

Currently the workaround is to simply put another instance of OctoPrint on startup using this command:

```
~/oprint/bin/python ~/OctoPrint/run --daemon start --port 5001 --pid /tmp/octoprint2 --basedir ~/.octoprint2
```

There is also a talk about this in OctoPrint: https://github.com/foosel/OctoPrint/issues/113#issuecomment-19679781

**Comment by LieutenantHawkeye** [comment]
I know this thread is over two years old now but I'm thinking about doing just this and have a question. If you create a second OctoPrint Instance how do you connect to it? Do you jsut go to http://octopi.local:5001 to access the second instance?

**Comment by guysoft** [comment]
@LieutenantHawkeye Yes, that should do it.

**Comment by LieutenantHawkeye** [comment]
@guysoft BTW do you just type that command into the terminal for the pi? Sorry I'm a newb at this kind of thing

**Comment by guysoft** [comment]
Yes, that would be a good start to see that it works. Youncan then look in
to how to make it start on boot.

**Comment by TGC-TECH** [comment]
know this is old but how would I make this command start at boot, right now I just replaced the restart octoprint command in octoprint settings with the one here so I can start it quickly without SSH but I would like some recomendation on how to go about enabling this command to autoluach at boot. Thanks!

**Comment by davr** [comment]
General advice for running commands on boot in linux is add the commands to
/etc/rc.local but no guarantee this is the best option for whatever you are
trying to do here

**Comment by guysoft** [comment]
You need to create a systemd service.
```

**Comment by enriqueiglesias** [comment]
General advice for running commands on boot in linux is add the commands to /etc/rc.local but no guarantee this is the best option for whatever you are trying to do here and for more queries you can visit **[HP Printer Support](http://hp.printertechsupportnumbers.com/)** and you can get the help from expert's.

**Comment by guysoft** [comment]
They new RaspberryPi 4 might actually revive this talk.

**Comment by SilverWingedHawk** [comment]
Is this issue still relevant ? Because you could leverage systemd template units (ie octoprint@.service, with the port as the instace "name", and a corresponding script to prepare for another configuration directory).
I'd be willing to work on a PR to get it done.
